aws-rds-dbcluster/src/main/java/software/amazon/rds/dbcluster/Translator.java

```diff
@@ -505,6 +508,7 @@ public static ResourceModel translateDbClusterFromSdk(
                 .domain(domain)
                 .domainIAMRoleName(domainIAMRoleName)
                 .enableCloudwatchLogsExports(dbCluster.enabledCloudwatchLogsExports())
+                .enableGlobalWriteForwarding(dbCluster.globalWriteForwardingRequested())
```

Comment:
Do we need to consider  GlobalWriteForwardingStatus as well?

Reply:
Adding stabilization checks.